docs/docsite/rst/community/collection_contributors/collection_integration_about.rst

```diff
@@ -0,0 +1,156 @@
+.. _collection_integration_tests_about:
+
+Understanding integration tests
+=================================
+
+.. note::
+
+  Some collections do not have integration tests.
+
+Integration tests are functional tests of modules and plugins.
+With integration tests, we check if a module or plugin satisfies its functional requirements. Simply put, we check that features work as expected and users get the outcome described in the module or plugin documentation.
+
+There are :ref:`two kinds of integration tests <collections_adding_integration_test>` used in collections:
+
+* integration tests that use Ansible roles
+* integration tests that use ``runme.sh``.
+
+This section focuses on integration tests that use Ansible roles.
+
+Integration tests check modules with playbooks that invoke those modules. The tests pass standalone parameters and their combinations, check what the module or plugin reports with the :ref:`assert <ansible_collections.ansible.builtin.assert_module>` module, and the actual state of the system after each task.
```

Comment:
```suggestion
Integration tests check modules with playbooks that invoke those modules. The tests pass standalone parameters and their combinations; check what the module or plugin reports with the :ref:`assert <ansible_collections.ansible.builtin.assert_module>` module, and the actual state of the system after each task.
```

Reply:
I think the comma is more appropriate here as it's all about what the tests do